docs/1.0.0/install_macos.md

```diff
@@ -0,0 +1,230 @@
+# Get your own RIF Lumino node up and running on MacOS
```

Comment:
this should be tested at some point; i created the following issue for this purpose: [RLP-955](https://jirainfuy.atlassian.net/browse/RLP-995)

Reply:
thxs